aip/general/0134.md

```diff
@@ -60,6 +60,12 @@ rpc UpdateBook(UpdateBookRequest) returns (Book) {
   - All remaining fields **should** map to URI query parameters.
 - There **should** be exactly one `google.api.method_signature` annotation,
   with a value of `"{resource},update_mask"`.
+- The operation should be [consistent][]: the completion of an update operation
```

Comment:
"strongly consistent"

Reply:
"read-after-write consistent" sounds odd to me. Probably would need to change the verb if we go with that.

Reply:
strongly consistent works here, will update.